tests/app/v2/notifications/test_post_notifications.py

```diff
@@ -515,9 +633,8 @@ def test_post_sms_notification_returns_400_if_not_allowed_to_send_int_sms(
     ]
 
 
-def test_post_sms_notification_with_archived_reply_to_id_returns_400(client, sample_template, mocker):
+def test_post_sms_notification_with_archived_reply_to_id_returns_400(client, sample_template):
     archived_sender = create_service_sms_sender(sample_template.service, "12345", is_default=False, archived=True)
-    mocker.patch("app.celery.provider_tasks.deliver_email.apply_async")
```

Comment:
Hmm I guess that was not necessary.. that makes sense as we are testing SMS in this function. 👍